packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1116,6 +1156,10 @@ abstract class SingleChildRenderObjectWidget extends RenderObjectWidget {
 /// storage for that child list, it doesn't actually provide the updating
 /// logic.)
 abstract class MultiChildRenderObjectWidget extends RenderObjectWidget {
+  /// Initializes this objects fields for its subclasses.
```

Comment:
I can't find any examples, but I think I used something likes this elsewhere.  I've fixed the grammar.  If we find more instances, we should make them the same.